requirements/prod.txt

```diff
@@ -113,6 +113,9 @@ tower==0.4.1
 # urllib3 is required by elasticsearch
 urllib3==1.10.3
 
+# django secure middleware
+django-secure==1.0.1
```

Comment:
We try to sort the packages in this file by name (alphabetically), can you move this line higher ? Also, no need to add "django secure middleware" comment.